airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -450,9 +450,13 @@ def collect(self):
             labels=["task_id", "dag_id", "execution_date"],
         )
         for task in get_landing_times():
-            task_duration_value = (
-                task.end_date - date_range(task.execution_date, num=2, delta=task.schedule_interval)[1]
-            ).total_seconds()
+            if task.schedule_interval:
```

Comment:
As per [tech standarization doc](http://readthedocs.jampp.com/tech-standardization/standardization/code/code_style/python.html#best-practices):

"Don’t do boolean checks when you mean “not None”. Boolean checks can invoke arbitrary and expensive code, and if what you mean is “not None” they’re not only slower but also wrong, since there are plenty of false yet not None objects. So if what you mean is `is not None`, write it like that."